### PR TITLE
Work around unremovable dir tree problem with some GNU packages

### PR DESCRIPTION
**Pull request by chdiza**

### CODE REVIEW

**Comment by DomT4** [comment]
`coreutils` is another, I think?

Thanks for picking this up @chdiza.

File: Library/Formula/coreutils.rb [hunk 1]
```diff
@@ -31,6 +31,16 @@ class Coreutils < Formula
   depends_on "gmp" => :optional
 
   def install
+    # Work around unremovable, nested dirs bug that affects lots of
+    # GNU projects. See:
+    # https://github.com/Homebrew/homebrew/issues/45273
+    # https://github.com/Homebrew/homebrew/issues/44993
+    # This is thought to be an el_captian bug:
+    # http://lists.gnu.org/archive/html/bug-tar/2015-10/msg00017.html
+    if MacOS::Version == :el_capitan
```
Comment (DomT4) [inline Library/Formula/coreutils.rb]: Tiny nit, but we tend to use `MacOS.version` *inside* formulae.

Comment (chdiza) [inline Library/Formula/coreutils.rb]: You're right.  I don't know why I did that.  I've never done that before.  I'll fix it!

**Comment by mikemcquaid** [comment]
@chdiza I'll fix it up myself but the `Whoops` commit would be great if it was one change per formula (makes squashing easier)

**Comment by mikemcquaid** [comment]
Thanks @chdiza!

**Pull request closed without merging**